CONTRIBUTING.md

```diff
@@ -1,66 +1,191 @@
+# Table of Contents
+
+<!-- toc -->
+- [Contributing to Deepchem](#contributing-to-deepchem)
+  - [Getting Started](#getting-started)
+  - [Pull Request Process](#pull-request-process)
+  - [Coding Conventions](#coding-conventions)
+  - [Documentation Conventions](#documentation-conventions)
+- [The Agreement](#the-agreement)
+- [Deepchem Technical Steering Committee](#deepchem-technical-steering-committee)
+<!-- tocstop -->
+
+## Contributing to DeepChem
+
+We actively encourage community contributions to DeepChem. The first
+place to start getting involved is
+[the tutorials](https://deepchem.readthedocs.io/en/latest/get_started/tutorials.html).
+Afterwards, we encourage contributors to give a shot to improving our documentation.
+While we take effort to provide good docs, there's plenty of room
+for improvement. All docs are hosted on Github, either in `README.md`
+file, or in the `docs/` directory.
+
+Once you've got a sense of how the package works, we encourage the use
+of Github issues to discuss more complex changes, raise requests for
+new features or propose changes to the global architecture of DeepChem.
+Once consensus is reached on the issue, please submit a PR with proposed
+modifications. All contributed code to DeepChem will be reviewed by a member
+of the DeepChem team, so please make sure your code style and documentation
+style match our guidelines!
+
+### Getting Started
+
+To develop DeepChem on your machine, we recommend using Anaconda for managing
+packages. If you want to manage multiple builds of DeepChem, you can make use of
+[conda environments](https://conda.io/projects/conda/en/latest/user-guide/concepts/environments.html)
+to maintain seperate Python package environments, each of which can be tied
+to a specific build of DeepChem. Here are some tips to get started:
+
+1. Fork the [DeepChem](https://github.com/deepchem/deepchem/) repository
+and clone the forked repository
+
+```bash
+git clone https://github.com/YOUR-USERNAME/deepchem.git
+cd deepchem
+```
+
+&nbsp;&nbsp;&nbsp;&nbsp; 1.1. If you already have DeepChem from source, update it by running
+```bash
+git fetch upstream
+git rebase upstream/master
+```
+
+2. Set up a new conda environment for DeepChem
+
+```bash
+conda create -n deepchem python=3.8
+conda activate deepchem
+```
+
+&nbsp;&nbsp;&nbsp;&nbsp; 2.1. DeepChem provides backend support with tensorflow, pytorch or jax.
+For developemnt of DeepChem with one of the above backend, update the environment using
+```bash
+conda env update -n deepchem --file requirements/backend-name/env_backend-name.yml --prune
+```
+You need to replace `backend-name` with one of `tensorflow`, `pytorch` or `jax`. PyTorch
+by default is installed with GPU support. For pytorch CPU, use the following command:
```

Comment:
By default, all environments are installed in CPU. If GPU support is required, then make sure CUDA is installed and then install the desired deep learning framework using the links below before installing deepchem inside the environment

1. tensorflow - just cuda installed
2. pytorch - https://pytorch.org/get-started/locally/#start-locally
3. jax - https://github.com/google/jax#pip-installation-gpu-cuda